client/app/license/spla/add/license-spla-add.html

```diff
@@ -37,7 +37,7 @@
         </div>
 
         <p class="text-danger"
-           data-ng-show="$ctrl.options.availableServers.length != null && !$ctrl.options.availableServers.length"
+           data-ng-if="$ctrl.options.availableServers.length != null && !$ctrl.options.availableServers.length"
```

Comment:
`data-ng-if="$ctrl.options.availableServers.length > 0"` ? At least one of those 2 conditions is useless

Reply:
the fact is, on init, `$ctrl.options.availableServers == null`, after loading, the same value is not null, but may have 0 servers available. (the second test is there to show error message if it is the case)

Reply:
But then it should be `$ctrl.options.availableServers != null && $ctrl.options.availableServers.length != 0` and not `$ctrl.options.availableServers.length != null && !$ctrl.options.availableServers.length`

Reply:
Yes didn't see the `availableServers.length != null`